src/main/webapp/js/index.js

```diff
@@ -12,10 +12,22 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/** Fetches all receipts from the server and adds them to the DOM. */
+async function displayAllReceipts() {
+  const params = new URLSearchParams();
+  params.append('isNewLoad', 'true');
```

Comment:
nit: It might be better to use a boolean here if it does not affect the servlet

Reply:
I also think it would be better to name this parameter something like getAllReceipts that describes what is actually being requested.

Reply:
@basaam0 I think params are always sent as a string, so even passing a boolean would require parsing it on the servlet

Reply:
@emilyvera this comment still seems unresolved? +1 to @avagarcia0 about the naming.

Reply:
@acheng1 I changed it in commit aece6b7 but I think when I merged another branch into this one it reverted back. What's a good workflow to avoid this from happening?

Reply:
That seems fine but you should have gotten merge conflicts in that case, sounds like you did not?

Reply:
I didn't get conflicts, it seems like it just overwrote it :/

Reply:
Oh wait...I just double checked and it's getAllReceipts :)

Reply:
Oh haha. Glad it worked!